win_configure/msi/wix/build_wxs_source.bat

```diff
@@ -0,0 +1,56 @@
+@echo off
+REM This script will generate Wix source files using Heat.exe toolset
+
+REM Defining needed variables 
+
+SET PBS_prefix=''
+@setlocal enableextensions enabledelayedexpansion
+set variable=%~dp0
+if "x!variable:~-29!"=="xpbspro\win_configure\msi\wix\" (
+	set variable=!variable:~0,-29!
+) else (
+@echo "Failed to parse PBS prefix location"
+goto theend
+)
+
+SET PBS_prefix=!variable!
+
+SET TOPDIR=%PBS_prefix%PBS
+
+if exist temp (
+@echo temp already available.. Deleting...
+rmdir /s /q temp
+)
+
+if not exist "%PBS_prefix%PBS\exec\etc\vcredist_x86.exe" (
+	@echo vcredist_x86.exe not available to build 
+	goto theend
+)
+
+
+@echo Creating source directory
+mkdir temp
+@if NOT %ERRORLEVEL% == 0 goto theend
+@echo done
+
+
+if exist %TOPDIR% (
+heat dir %TOPDIR%\exec -ag -cg pbsproexec -sfrag -sreg -template fragment -out .\temp\pbsproexec.wxs -directoryid "PBS_PRO" -dr "INSTALLFOLDER" -var var.pbsproexec
+@if NOT %ERRORLEVEL% == 0 goto theend
+@echo done
+)
+
+if exist %TOPDIR% (
+heat dir %TOPDIR%\home -ag -cg pbsprohome -sfrag -sreg -template fragment -out .\temp\pbsprohome.wxs -dr "INSTALLFOLDER" -var var.pbsprohome
```

Comment:
Idea behind why I left those dynamic files:
In case of minor change in PBS code, developer can build it and can use the existing wxs source files. This will save the time of second time generation of Wix source files